build-tests/api-documenter-test/etc/api-documenter-test.api.json

```diff
@@ -192,11 +192,11 @@
               "text": "export interface Constraint "
             }
           ],
+          "extendsTokenRanges": [],
```

Comment:
In a future PR, I'd like to never write empty token ranges to API doc models.

Reply:
The serializer currently takes the approach of always writing every field, even if the contents are empty. The original motivation was to make the JSON file format somewhat self-documenting, such that someone could write a correct loader for it without having to look at the source code.  It seemed more important back then, because there was no `api-extractor-model` library to validate the deserialization, just a JSON schema and some TypeScript interfaces to access the JSON objects.

We could consider a different approach of trying to make the .api.json files as small as possible.  But that would only make sense if we applied it across all fields, not just the token ranges.